packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1574,3 +1574,22 @@ class SliverCrossAxisGroup extends MultiChildRenderObjectWidget {
     return RenderSliverCrossAxisGroup();
   }
 }
+
+/// A sliver that places multiple sliver children in a linear array along
+/// the main axis, one after another.
+///
+/// For pinned sliver children, the behavior is that the pinned sliver should
```

Comment:
Can you reference the API you are referring to here? Folks may not know what a pinned sliver child is in this context. Some sample code of this exact case would be really great actually. :)